theory.md

```diff
@@ -2,5 +2,7 @@
 Here you can find all information pertaining to the theory applied for the implementation of this library
 
 ### Table of contents
-- [Chapter 1 - Solvers](theory/chapter1.md)
+- [Chapter1-Random Numbers and Monte Carlo Integration](theory/Chapter1-Random Numbers and Monte Carlo Integration.md)
+- [Chapter2-Fourier Series and Karhunen Loeve Expansions](theory/Chapter2-Fourier Series and Karhunen Loeve Expansions.md)
+- [Chapter3-Spectral Representation of Random Processes](theory/Chapter3-Spectral Representation of Random Processes.md)
```

Comment:
The files must be renames so that they contain no spaces. Otherwise they cannot be displayed.

Reply:
done, https://github.com/odyred/MSolve.Stochastic/blob/gh-pages/theory.md